CSharp/BatchExplorer/README.md

```diff
@@ -1,5 +1,7 @@
 ## Azure Batch Explorer
 
+**Note:** For a supported up-to-date cross-platform client tool to browse Azure Batch resources see [Batch Labs](https://github.com/Azure/BatchLabs). Batch Explorer will continue to exist in this repository only as a sample and will not be updated to expose the latest features of Azure Batch. We will not be accepting new feature contributions.
```

Comment:
How about: 

[Batch Labs](https://github.com/Azure/BatchLabs) is a cross-platform client tool for browsing Azure Batch resources. Batch Labs is supported by Microsoft and is the preferred tool for use with Azure Batch.  

Batch Explorer will continue to exist in this repository only as a sample and will not be updated to expose the latest features of Azure Batch. We will not be accepting new feature contributions.